pipeline_config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "RapidProBaseURL": "https://textit.in",
+  "RapidProBaseURL": "textit.in",
```

Comment:
I don't think this is a valid URL any more if we do this. 

Consider RapidProBaseDomain

Reply:
Renamed to RapidProDomain